internal/controller/stages/stages.go

```diff
@@ -851,6 +854,14 @@ func (r *reconciler) syncNormalStage(
 		return status, nil
 	}
 
+	// TODO: Remove this when we have added support for multiple Freight below.
+	if len(stage.Spec.RequestedFreight) > 1 {
+		logger.Info(
+			"Stage requests multiple Freight. Auto-promotion support has however not been implemented for this (yet).",
+		)
+		return status, nil
+	}
```

Comment:
👍 This will be fun to figure out. Glad to take it out of the equation temporarily.